### ISSUE TITLE
DelimitedList with empty string return ['']

### ISSUE THREAD

**Issue by TTWShell** [marshmallow-code/webargs#623]
https://github.com/marshmallow-code/webargs/blob/ed8840c4bb06f1b3bcaf664e025183423349d9d2/src/webargs/fields.py#L72

```
In [8]: from webargs.fields import DelimitedList, Str

In [9]: dlist = DelimitedList(Str(), required=True)

In [10]: dlist.deserialize('')
Out[10]: ['']
```

Excepted empty list `[]` ?

**Comment by lafrech** [comment]
I believe you're right. Would you like to propose a fix?

**Comment by TTWShell** [comment]
> I believe you're right. Would you like to propose a fix?

I will do it

**Comment by sirosen** [comment]
This was fixed in #624 and released. (Thanks @lafrech for doing that!)
